spec/Stratum.md

```diff
@@ -0,0 +1,137 @@
+Stratum is a light-weight mining protocol.
+
+# List of methods
+
+ * [mining.subscribe](#mining.subscribe)
+ * [mining.authorize](#mining.authorize)
+ * [mining.notify](#mining.notify)
+ * [mining.submit](#mining.submit)
+
+# Specification
+
+## mining.subscribe
+
+Used for subscribing mining jobs.
+
+Params: No parameters
+
+Return Type: No parameters
+
+Request Example
+```
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "mining.subscribe",
+    "params": []
+}
+```
+
+Response Example
+```
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "result": [],
+    "error": null
+}
+```
+
+## mining.authorize
+
+Used for authorizing miner.
+
+Params:
+ 1. name: `string`
+ 2. password: `string`
+
+Return Type: `bool`
+
+Request Example
+```
+{
+    "jsonrpc": "2.0",
+    "id":2,
+    "method": "mining.authorize",
+    "params": ["miner1", "password"]
+}
+```
+
+Response Example
+```
+{
+    "jsonrpc": "2.0",
+    "id": 2,
+    "result": true,
+    "error": null
+}
+```
+
+## mining.notify
+
+Used for sending notifications with mining jobs
```

Comment:
Used for sending notifications with mining jobs -> Used for sending notifications regarding mining jobs.